docs/en/latest/plugins/ldap-auth.md

```diff
@@ -41,12 +41,13 @@ This authentication plugin use [lualdap](https://lualdap.github.io/lualdap/) plu
 
 ## Attributes
 
-| Name     | Type   | Requirement | Default | Valid | Description                                                                                                                                                      |
-| -------- | ------ | ----------- | ------- | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| base_dn | string | required    |         |       | the base dn of the `ldap` server (example : `ou=users,dc=example,dc=org`)   |
-| ldap_uri | string | required    |         |       | the uri of the ldap server  |
-| use_tls | boolean | optional    |    `true`     |       | Boolean flag indicating if Transport Layer Security (TLS) should be used. |
-| uid | string | optional    |     `cn`      |     | the `uid` attribute |
+| Name     | Type    | Requirement | Default | Valid |                                                                                                           |Description                                                                                                                                                      |
+| -------- | ------- | ----------- | ------- | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| base_dn  | string  | required    |         |       | the base dn of the `ldap` server (example : `ou=users,dc=example,dc=org`)                                 |
+| ldap_uri | string  | required    |         |       | the uri of the ldap server                                                                                |
+| use_tls  | boolean | optional    | `true`  |       | Boolean flag indicating if Transport Layer Security (TLS) should be used.                                 |
+| uid      | string  | optional    | `cn`    |       | the `uid` attribute                                                                                       |
+| user_dn  | string  | required    |         |       | the user dn of the `ladp` client (example: `cn=user01,ou=users,dc=example,dc=org`)                        |
```

Comment:
Hi @zaunist, thanks to pick this issue! Here has one sample from https://apisix.apache.org/docs/apisix/plugins/key-auth#attributes , and it splits attributes into 2 parts: For Consumer and For Route/Service, please take a look 😄

Reply:
thx, I will update later.